jax/experimental/jax2tf/jax_export.py

```diff
@@ -154,6 +158,67 @@ class Exported:
         starting with the primal arguments and followed by a cotangent argument
         for each primal output. It returns a tuple with the cotangents
         corresponding to the flattened primal inputs.
+
+  Calling convention for the exported module:
+
+  The `mlir_module` has a `main` function that takes an optional first
+  platform index argument if the module supports multiple platforms
+  (`len(lowering_platforms) > 1`), followed by the kept array arguments
+  (corresponding to `module_kept_var_idx` and `in_avals`).
+  The platform index is a i32 scalar encoding the index of the current
+  compilation platform into the `lowering_platforms` sequence.
+
+  Inner functions use a different calling convention: an optional
+  platform index argument, optional dimension variable arguments specified
+  using scalar tensors of type i32 or i64,
+  followed by optional token arguments (in presence of side effects),
+  followed by the regular array arguments.
+  The dimension arguments correspond to the dimension variables appearing in
+  the `args_avals`, in sorted order.
+
+  Consider the lowering of a function with one array argument of type "f32[w,
+  2 * h]", where "w" and "h" are two dimension variables.
+  Assume that we use multi-platform lowering, and we have
+  ordered effects. The `main` function will be as follows:
+
+      func public main(platform_index: i32, arg: f32[?, ?]) {
+         arg_w = hlo.get_dimension_size(arg, 0)
+         dim1 = hlo.get_dimension_size(arg, 1)
+         arg_h = hlo.floordiv(dim1, 2)
+         call _check_shape_assertions(arg)  # See below
+         token = new_token()
+         token_out, res = call _wrapped_jax_export_main(platform_index, arg_h, arg_w, token_in, arg)
```

Comment:
I did not get why the order `arg_w`, `arg_h` need change to 'arg_h', 'arg_w' here ? Any special reason or just typo ?
Thanks